src/main/kotlin/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacket.kt

```diff
@@ -41,9 +41,12 @@ import org.jitsi.rtp.util.getShortAsInt
 // Valid values are 0 (packet wasn't received), 1 or 2.
 typealias DeltaSize = Int
 
-class ReceivedPacket(val seqNum: Int, val deltaTicks: Short) {
+class ReceivedPacket(val seqNum: Int, val deltaTicks: Short, val received: Boolean = true) {
```

Comment:
if i'm remembering right (and based on the same) i think the idea here was to specifically model a 'received packet' (as opposed to one that wasn't received), so at the very least i think maybe we'd want a rename of this class if we're going to change the semantics of its meaning.

Reply:
also, are we sure we're not using `packets_.size` anywhere to get a quick number on the total number of received packets or something?

Reply:
I agree that the name ReceivedPacket is weird now that it has a `received` boolean, but the comments in this code said it was modeled on the equivalent code in libwebrtc, and libwebrtc did the same thing: (https://webrtc.googlesource.com/src/+/b468616a695caef5e9ae40c3aac634d06cea4f85%5E%21/#F5).

I don't see any use of `packets_.size` anywhere.

Reply:
> I agree that the name ReceivedPacket is weird now that it has a `received` boolean, but the comments in this code said it was modeled on the equivalent code in libwebrtc, and libwebrtc did the same thing: (https://webrtc.googlesource.com/src/+/b468616a695caef5e9ae40c3aac634d06cea4f85%5E%21/#F5).

Ah, interesting.  Is it a direct port or merely 'inspired by'?  If it's a direct port we can keep it, if not I wonder if we might be better off keeping it more clear, but I dunno.

> I don't see any use of `packets_.size` anywhere.

that's good, i worry it could be an easy mistake to make though in the future.